icu4j/main/tests/core/src/com/ibm/icu/dev/data/numberformattestspecification.txt

```diff
@@ -1668,15 +1668,39 @@ pattern	lenient	parse	output	breaks
 0E0'.'x	1	5E3.x	5000
 0E0'.'x	0	5E3.x	5000
 
-test parse lowercase currency
+test parse() lowercase currency
+set locale en
+set pattern ¤¤0
+set currency USD
+begin
+parse	output	breaks
+USD123	123
+USD 123	123	HK
+usd123	123	CK
+usd 123	123	CHK
+Usd123	123	CK
+Usd 123	123	CHK
```

Comment:
Steven has a comment about the mix of upper and lower case.
https://unicode-org.atlassian.net/browse/ICU-13696
"there will never be a `USd` vs `usd` vs `USD` that would be expected to be different."
Is it a problem?